app/Http/ViewComposers/ProfileComposer.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\ViewComposers;
+
+use Illuminate\View\View;
+// use App\Repositories\Contracts\OptionRepository;
+use App\Eloquent\Option;
+
+class ProfileComposer
+{
+
+    /**
+     * Create a new profile composer.
+     *
+     * @return  void
+     */
+    protected $option;
+
+    public function __construct(
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Classes.PropertyDeclaration.Multiple</br>**Severity**: error</br>**File**: app/Http/ViewComposers/ProfileComposer.php</br>**Line**: 21</br>There must not be more than one property declared per statement